source/_cookbook/SSL guide.md

```diff
@@ -0,0 +1,570 @@
+
+#### {% linkable_title 5 - check the incoming conection %}
+
+<p class='note'>
+Following on from Step 4 your SSH will still be in the certbot folder.  If you edit your configuration files over SSH you will need to change to your homeassistant folder:
+
+```bash
+$ cd ~/.homeassistant
+```
+
+If you use samba shares to edit your files you can exit your SSH now.
+</p>
+
+Go to your router's configuration pages and set up a new port forwarding rule, thus:
+
+```text
+Service name - ha_ssl
+Port Range - 443
+Local IP - YOUR-HA-IP
+Local Port - 8123
+Protocol - Both
+```
+
+If during step 4 you had to use port 443 instead of port 80 to generate your certificate, you should delete that rule now.
+
+Remember to save the rule changes.
+
+Now edit your configuration.yaml file to reflect the SSL entries and your base URL (changing the examplehome subdomain to yours):
+
+```yaml
+http:
+  api_password: YOUR_PASSWORD
+  ssl_certificate: /etc/letsencrypt/live/examplehome.duckdns.org/fullchain.pem
+  ssl_key: /etc/letsencrypt/live/examplehome.duckdns.org/privkey.pem
+  base_url: examplehome.duckdns.org
+```
+  
+You may wish to set up other options for the http component at this point, these extra options are beyond the scope of this guide but can be found on the http component page here: [http](https://home-assistant.io/components/http/)
+
+Save the changes to configuration.yaml.  Restart HA.
+
+In step 3 we accessed our HA from the outside world with our DuckDNS URL and our port number.  We are going to use a slightly different URL this time.
+
+```text
+https://examplehome.duckdns.org
+```
+
+Note the S after http, and that no port number is added.  This is because https will use port 443 automatically, and we have already set up our port forward to redirect this request to our HA instance on port 8123.
+
+You should now be able to see your HA instance via your DuckDNS URL, and importantly note that your browser shows the connection as secure.
+
+You will now NO LONGER be able to access your HA via your old internal IP address in the way you previously have.  Your default way to access your HA instance, even from inside your house, is to use your DuckDNS URL.
+
+In cases where you need to access via the local network only (which should be few and far between) you can access it with the following URL (note the added S after http):
+
+```text
+https://YOUR-HA-IP:8123
+```
+
+...and accepting the browsers warning that you are connecting to an insecure site.  This warning occurs because your certificate expects your incoming connection to come via your DuckDNS URL.  It does not mean that your device has suddenly become insecure.
+
+Some cases such as this are where your router does not allow 'loopback' or where there is a problem with incoming connections due to technical failure.  In these cases you can still use your internal connection and ignore the warnings.

```

Comment:
It's again about the loopback. Which I don't know how to explain in two or less sentences 😄